modules/stars.py

```diff
@@ -20,8 +20,280 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
+from typing import Optional
+
+import asyncpg
+import discord
+from datetime import datetime as date
```

Comment:
Why not just import datetime to keep consistency with other modules? It's only used in one place.